bioptim/limits/constraints.py

```diff
@@ -443,6 +445,34 @@ def inter_phase_continuity(ocp):
             pt.list_index = -1
             pt.add_or_replace_to_penalty_pool(ocp, ocp.nlp[pt.phase_pre_idx])
 
+    @staticmethod
+    def inter_node_continuity(ocp):
```

Comment:
I don't think continuity is the proper word... equality?
Nor that "inter" make much sense as it is nececessarily "inter_node" (what would mean intra_node?)

Reply:
then go for node_equalities?